src/coreComponents/physicsSolvers/SolverBase.cpp

```diff
@@ -401,14 +426,14 @@ real64 SolverBase::setNextDtBasedOnNewtonIter( real64 const & currentDt )
   if( newtonIter < iterIncreaseLimit )
   {
     // Easy convergence, let's increase the time-step.
-    nextDt = currentDt * m_nonlinearSolverParameters.timeStepIncreaseFactor();
+    nextDt = round( currentDt * m_nonlinearSolverParameters.timeStepIncreaseFactor() );
```

Comment:
this is experimental, right?

Reply:
Yes, it's not the main purpose of this PR. If the time-steps are larger than O(10) rounding shouldn't be an issue, right?

Reply:
how about < 0.5 ? or i am missing something...

Reply:
Yes, then this doesn't work

Reply:
yeah, we should not use round here. For some cases we need very tiny time steps.